src/inmanta/plugins.py

```diff
@@ -219,6 +225,78 @@ def type_string_internal(self) -> str:
     None: Null(),  # Only NoneValue will pass validation
 }
 
+python_to_model = {
+    str: inmanta_type.String(),
+    float: inmanta_type.Float(),
+    numbers.Number: inmanta_type.Number(),
+    int: inmanta_type.Integer(),
+    bool: inmanta_type.Bool(),
+    dict: inmanta_type.LiteralDict(),
+    list: inmanta_type.LiteralList(),
+}
+
+
+def to_dsl_type(python_type: type[object]) -> inmanta_type.Type:
```

Comment:
How about `Unknown`? Do we want to support that already? i.e. `int | Unknown` rather than `int` with `allow_unknowns=True`. Or is that fine for iso8-only. I expect we only use that in std, where we have already branched off anyway?

Reply:
That is for later, but good remark